packages/website/src/examples/layout/Table/Table.demo.tsx

```diff
@@ -19,36 +20,42 @@ interface IExampleRowData {
     body: string;
 }
 
+const columnHelper = createColumnHelper<IExampleRowData>();
+
+/**
+ * Define your columns outside the component rendering the table
+ * (or memoize them) to avoid unnecessary render loops
+ */
+const columns: Array<ColumnDef<IExampleRowData>> = [
+    columnHelper.accessor('userId', {
+        header: 'User ID',
+        cell: (info) => info.row.original.userId,
+    }),
+    columnHelper.accessor('id', {
+        header: 'Post ID',
+        cell: (info) => info.row.original.id,
+    }),
+    columnHelper.accessor('title', {
+        header: 'Title',
+        cell: (info) => info.row.original.title,
+    }),
+    Table.CollapsibleColumn as ColumnDef<IExampleRowData>,
+    // or if you prefer an accordion behaviour
+    // Table.AccordionColumn as ColumnDef<IExampleRowData>,
+];
+
 export default () => {
-    const columnHelper = createColumnHelper<IExampleRowData>();
-    const columns: Array<ColumnDef<IExampleRowData>> = [
-        columnHelper.accessor('userId', {
-            header: 'User ID',
-            cell: (info) => info.row.original.userId,
-        }),
-        columnHelper.accessor('id', {
-            header: 'Post ID',
-            cell: (info) => info.row.original.id,
-        }),
-        columnHelper.accessor('title', {
-            header: 'Title',
-            cell: (info) => info.row.original.title,
-        }),
-        Table.CollapsibleColumn as ColumnDef<IExampleRowData>,
-        // or if you prefer an accordion behaviour
-        // Table.AccordionColumn as ColumnDef<IExampleRowData>,
-    ];
-    const [data, setData] = useState([]);
-    const [loading, setLoading] = useState(false);
+    const [data, setData] = useState(null);
+    const [loading, setLoading] = useState(true);
     const [pages, setPages] = useState(1);
 
-    const fetchData = (state: any) => {
+    const fetchData: onTableChangeEvent = async (state) => {
         setLoading(true);
         const searchParams = new URLSearchParams({
             _sort: state.sorting?.[0]?.id ?? 'userId',
             _order: state.sorting?.[0]?.desc ? 'desc' : 'asc',
-            _page: state.pagination.pageIndex + 1,
-            _limit: state.pagination.pageSize,
+            _page: '' + state.pagination.pageIndex + 1,
+            _limit: '' + state.pagination.pageSize,
```

Comment:
I would prefer if we called toString directly
```suggestion
            _page: (state.pagination.pageIndex + 1).toString(),
            _limit: state.pagination.pageSize.toString(),
```